client/state/google-apps-users/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External Dependencies
+ */
+
+import { combineReducers } from 'redux';
+import uniqBy from 'lodash/uniqBy';
+import property from 'lodash/property';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	GOOGLE_APPS_USERS_FETCH,
+	GOOGLE_APPS_USERS_FETCH_COMPLETED,
+	GOOGLE_APPS_USERS_FETCH_FAILED,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+export function items( state = [], action ) {
+	switch ( action.type ) {
+		case GOOGLE_APPS_USERS_FETCH_COMPLETED:
+			return uniqBy( state.concat( action.items ), property( 'email' ) );
+		case SERIALIZE:
+			return [];
+		case DESERIALIZE:
+			return [];
+	}
+	return state;
+}
+
+export function loaded( state = false, action ) {
+	switch ( action.type ) {
+		case GOOGLE_APPS_USERS_FETCH:
+		case SERIALIZE:
+		case DESERIALIZE:
+			return false;
```

Comment:
Do we ever re-fetch? On a subsequent fetch request, this would flip the `loaded` state to false, when in fact we may have some usable items.